test/e2e/networking.go

```diff
@@ -83,7 +83,6 @@ var _ = framework.KubeDescribe("Networking", func() {
 
 		By("checking kube-proxy URLs")
 		config.GetSelfURL(ports.ProxyHealthzPort, "/healthz", "200 OK")
-		config.GetSelfURL(ports.ProxyStatusPort, "/proxyMode", "iptables") // the default
```

Comment:
I mean we can already check kube-proxy process's status by visiting `/healthz` endpoint. Why do we need to visit `proxyMode` endpoint again?